services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Movie, Genre, Actor
+from django.db.models.query import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(genres__id__in=genres_ids,
+                               actors__id__in=actors_ids)

```

Comment:
This if statement is redundant. Check for filter works. It works as and:
movies.filter(genres__id__in=genres_ids).filter(actors__id__in=actors_ids)
will be the same as movies.filter(genres__id__in=genres_ids, actors__id__in=actors_ids)